src/components/TimeSeriesCard/TimeSeriesCard.jsx

```diff
@@ -486,11 +451,13 @@ const TimeSeriesCard = ({
     >
       {!isChartDataEmpty ? (
         <>
-          <LineChartWrapper
-            size={newSize}
-            isEditable={isEditable}
-            isExpanded={isExpanded}
-            numberOfPoints={valueSort && valueSort.length}
+          <div
+            className={classNames(`${iotPrefix}--time-series-card--wrapper`, {
+              [`${iotPrefix}--time-series-card--wrapper--expanded`]: isExpanded,
```

Comment:
Our BEM style is so convoluted so this is not as clear as it should be but I think we should have
`${iotPrefix}--time-series-card--wrapper--expanded` be `${iotPrefix}--time-series-card--wrapper__expanded`.
Same thing with the editable modifier
`${iotPrefix}--time-series-card--wrapper--editable` should be `${iotPrefix}--time-series-card--wrapper__editable`

I am not so hung up on this but figure I would point it out.

Reply:
Yeah that makes sense to me. Do we have an issue to fix that across the library?